site/src/pages/WorkspacePage/WorkspaceReadyPage.tsx

```diff
@@ -100,13 +100,11 @@ export const WorkspaceReadyPage = ({
       ["canceling", "deleting", "pending", "starting", "stopping"].includes(
```

Comment:
CLI allows to enable build options on update/start/create workspace. Do we need parity in the WebUI?

Reply:
I think so. I will think about how we can make that.

Reply:
Would make sense to add a new section "Ephemeral parameters" as we did for the "Immutable parameters"?

Reply:
Yes. That's something I had in mind, but you may want to consult it with Ben. Maybe there is a better view idea.

Reply:
Yeah, would be open to also having it in the template settings page there. Probably not something we should show on the "create workspace" page though.

Reply:
> Add ephemeral parameters on the template settings page

I'm not sure about this point...

> Add ephemeral parameters on the workspace parameters page

Yes

> Probably not something we should show on the "create workspace" page though.

I will trim it also on the CLI side.